runtime/doc/vim.man

```diff
@@ -36,11 +36,11 @@ DESCRIPTION
             vim [options] [filelist]
 
        If the filelist is missing, the editor will start with an empty buffer.
-       Otherwise  exactly  one out of the following four may be used to choose
+       Otherwise exactly one out of the following four may be used  to  choose
```

Comment:
The spacing changes look odd to me: I’m not terribly familiar with roff semantics, but it seems like we removed some instances of 2 spaces between words but then added new instances throughout this file?

If one space in roff means an inter-word space, let’s stick to consistently one space between words.

(I guess it’s also possible it’s a different kind of invisible character I can’t see; a remark on why would be helpful for me in that case.)

Reply:
I noticed those to. However I think those `.man` files are generated automatically using the Makefile (and probably some vimscript vodoo. So I am not too worried about it.

Reply:
I was trying to figure this out too, why text with different spacing is generated each time.
The command that is written in Makefie is as follows:
`nroff -man $< | sed -e s/.//g > $@`.
The pattern contains a tab character.
Started reading the [documentation](https://heirloom.sourceforge.net/doctools/troff.pdf), realised that it is possible, probably can be configured, but I no longer have enough time and patience.

Reply:
Sorry, missed the auto-generation! Thanks both. (I think you meant a backspace, not tab, but close enough for these purposes.)

Reply:
Yes, of course. Thank you.